doc/website/sidebars.js

```diff
@@ -391,15 +391,19 @@ module.exports = {
                     label: 'GORM',
                     link: {
                         type: 'doc',
-                        id: 'guides/orms/gorm',
+                        id: 'guides/orms/gorm/gorm',
```

Comment:
yucky url, prefer the root (old URL for GORM guide) will be the category home, and the getting started will be named getting-started